src/app/user-page/user-page.component.html

```diff
@@ -13,6 +13,9 @@
       <img mat-card-image [src]="user?.avatarUrl" class="mt-0 mx-auto mb-3" alt="Dockstore avatar" />
       <mat-card-content fxLayout="column" fxLayoutAlign="center center">
         <h4 matLine class="accent-1-dark mt-2 mb-0">{{ user?.username }}</h4>
+        <a href="{{ gitHubProfile?.link }}" target="_blank" class="accent-1-dark">
```

Comment:
You should add `rel="noopener noreferrer"`, like (almost?) all our other external links.